samplefuncs.py

```diff
@@ -70,7 +70,7 @@ def TopicTargetList(topic=None, data=None, srv=None):
             targets.append('log:warn')
 
     return targets
-
+	
```

Comment:
In future, please use one PR to solve one thing. Samplefuncs is not part of your PR so don't touch it. It's ok now, don't change this, but kindly keep in mind for any PR

Reply:
Yeah I had altered it, to show how I incuded an image call from a regular message, but @sumnerboy12 suggested I leave it out as my examples were....so this is why samplefuncs has been touched in the pr (first I changed, then I removed my changes).